cmd/harmony/main.go

```diff
@@ -223,6 +225,36 @@ func initSetup() {
 	hmykey.SetHmyPass(myPass)
 }
 
+func setUpConsensusKey(nodeConfig *nodeconfig.ConfigType) {
+	// If FN node running, they should either specify blsPrivateKey or the file with passphrase
+	if *blsKeyFile != "" && *blsPassFile != "" {
+		passPhrase, err := utils.ReadStringFromFile(*blsPassFile)
```

Comment:
Adapt this to use `GetPassphraseFromSource`:

```suggestion
		passPhrase, err := utils.GetPassphraseFromSource(*blsPass)
```